api/src/emails/cle/classe.js

```diff
@@ -123,6 +123,28 @@ module.exports = (emailsEmitter) => {
       capture(error);
     }
   });
+
+  // Notify admin CLE Verif
+  emailsEmitter.on(SENDINBLUE_TEMPLATES.CLE.CLASSE_NOTIFY_VERIF, async (classe) => {
```

Comment:
il faudrait trouver un moyen de coverage ce code, beaucoup de maping

Reply:
ouais je me le suis noté dans un coin de ma tete ;)